Source/Charts/Data/Interfaces/IValueSliceLineChartDataSet.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  IValueSliceLineChartDataSet.swift
+//  Charts
+//
+
+import Foundation
+import CoreGraphics
+
+@objc
+public protocol IValueSliceLineChartDataSet: ILineChartDataSet
+{
+
+    /// If true, drawing circles for highlighted values is enabled
+    var drawCirclesOnlyForHighlightedValueEnabled: Bool { get set }
```

Comment:
Following the example of `BarChartDataProvider`, you shouldn't have this boolean available